docs/architecture/self-hosted/download-logs.md

```diff
@@ -8,12 +8,9 @@ tags:
   - support logs
 ---
 
-As an admin user, you can download your environment logs from Prophecy without needing access to your Prophecy cluster or the assistance of Prophecy Support. This reduces the delay in debugging any issues with your Prophecy services.
+Prophecy lets you download logs to help debug issues with your Prophecy services. As an admin, you can download environment logs directly from Prophecy without needing access to your Prophecy cluster or assistance from Prophecy Support.
 
```

Comment:
@kathweinschenkprophecy would you mind copying this image to use here as well? https://docs.prophecy.io/release_notes/2024/November_2024/#download-logs

I know we don't like to duplicate content, but a picture really is needed here.

Reply:
yes I'll just direct it to that same image!